Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM registry.redhat.io/3scale-amp2/system-rhel8:3scale2.15
-
+FROM registry.redhat.io/3scale-amp2/system-rhel9:3scale2.16
```

Comment:
NOTE: This image is not available  yet, for local testing replace with an alternative.